tests/test_result_conversions.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+
+import pandas as pd
+
+from vortexasdk.result_conversions import format_datatypes
+
+
+def test_format_datatypes():
+    # Create a sample DataFrame with different timestamp formats
+    data = {
+        "date_col_dont_convert": ["2023-10-01", "2023-10-02", "2023-07-01"],
+        "date_col_timestamp": ["2023-10-01", "2023-10-02", "2023-07-01"],
+        "mixed_iso_format_timestamps": [
+            "2023-10-03 12:00:00",
```

Comment:
These aren’t ISO format date times. If they’re missing the T. What are these test cases supposed to be?

Reply:
Handled in slack. They’re formats other than 8601. I assumed we stuck with the latest, but that’s not necessarily true.